reftools/tests/test_get_phot_pars.py

```diff
@@ -141,13 +132,12 @@ def test_make_row_struct_2(self):
             6.7696605688248e-14, 5.778989112976214e-14, 6.871738863125632e-14,
             6.430043639034794e-14, 5.2999039030883145e-14]
         assert_allclose(rd['results'], ans)
-        assert_allclose(
-            rd['parvals'],
-            [[4824, 4868.2, 4912.4, 4956.6, 5000.8, 5045, 5089.2, 5133.4,
-              5177.6, 5221.8, 5266],
-             [52334, 53919, 53920, 55516]])
+        assert len(rd['parvals']) == 2
```

Comment:
Numpy doesn't like you comparing ragged array anymore, so have to split the test.